client/src/components/Map.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
+
+function Map({ LocationMarker, stations, fuelValue }) {
+  const mapToken = process.env.REACT_APP_API_KEY;
+
+  return (
+    <MapWrapper center={[52.500478, 13.376696]} zoom={13} scrollWheelZoom={true}>
+      <TileLayer
+        url={
+          'https://api.mapbox.com/styles/v1/alexf090/cl2uhl8gx008p14qop192uczn/tiles/256/{z}/{x}/{y}@2x?access_token=' +
+          mapToken
+        }
+      />
+      {stations
+        .filter(station => station.fuelPrices[fuelValue] !== null)
```

Comment:
This is fine, but if you want to make it even more readable, you could create a variable with your filtered results first :octocat: